test/api/activation.unit.js

```diff
@@ -7,14 +7,15 @@
 //      Upstream: https://github.com/mochajs/mocha/issues/185
 process.env.NODE_ENV = 'test';
 
-const chai = require('chai');
-const expect = require('chai').expect;
+import { expect } from 'chai';
```

Comment:
I just saw https://evertpot.com/using-top-level-await-is-bc-break/ which refers to Node growing support for `require`ing an ES module in some cases. Which, if it works, would make this PR unnecessary.